lib/chef/resource/aws_route_table.rb

```diff
@@ -58,6 +58,8 @@ class Chef::Resource::AwsRouteTable < Chef::Provisioning::AWSDriver::AWSResource
   #
   attribute :routes, kind_of: Hash
 
+  attribute :ignore_route_targets, kind_of: Array, default: []
```

Comment:
Can you add some RDoc here that explains what users are expected to provide in their cookbooks, and how that will change their route table management?